qiskit/providers/ibmq/utils/converters.py

```diff
@@ -12,22 +12,21 @@
 # copyright notice, and modified files need to carry a notice indicating
 # that they have been altered from the originals.
 
-"""Converters
-"""
+"""Utilities related to conversion."""
 
+from typing import Union
 import datetime
 import dateutil
 
 
-def utc_to_local(utc_dt):
-    """Takes a UTC datetime object or string and
-    converts it to a local timezone datetime.
+def utc_to_local(utc_dt: Union[datetime, str]) -> datetime:
```

Comment:
We should include this function in the public docs since it could be quite useful to the users. You'll need to create a `utils.rst` file in `apidocs`, add it to `ibmq.rst`, and list the function in `__init__`.